src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneStoppedNode.kt

```diff
@@ -107,6 +108,11 @@ internal data class StandaloneStoppedNode(
         ssh: SshConnection,
         threadDump: ThreadDump
     ) {
+        if (skipStartupWait) {
+            logger.info("Skipping Jira startup wait")
+            return
+        }
+        logger.info("Waiting for Jira node to start...")
```

Comment:
Logging might be too verbose? Would `debug` be more appropriate or `info` is fine?

I pushed just what I had for now. Am willing to change when decided.

Reply:
Applied this suggestion